src/include/duckdb/storage/buffer/buffer_pool.hpp

```diff
@@ -86,12 +87,24 @@ class BufferPool {
 	atomic<idx_t> maximum_memory;
 	//! Eviction queue
 	unique_ptr<EvictionQueue> queue;
-	//! Total number of insertions into the eviction queue. This guides the schedule for calling PurgeQueue.
-	atomic<uint32_t> queue_insertions;
 	//! Memory manager for concurrently used temporary memory, e.g., for physical operators
 	unique_ptr<TemporaryMemoryManager> temporary_memory_manager;
 	//! Memory usage per tag
 	atomic<idx_t> memory_usage_per_tag[MEMORY_TAG_COUNT];
+
+	//! We trigger a purge of the eviction queue every INSERT_INTERVAL insertions
+	constexpr static idx_t INSERT_INTERVAL = DEFAULT_STANDARD_VECTOR_SIZE;
```

Comment:
`INSERT_INTERVAL` is now defined twice, once here and once in `BufferPool::AddToEvictionQueue` - with different values. Might be causing some confusion?